src/registrar/migrations/0039_merge_20231013_2029.py

```diff
@@ -0,0 +1,12 @@
+# Generated by Django 4.2.1 on 2023-10-13 20:29
```

Comment:
**comment (blocking):** I know, I know migrations are the bane of our existence 🤣 However, I don't think we should be merging migrations files and each migration should be it's own separate migration.

I think this happened when you merged a PR from main into your working branch and bc the migration numbers were the same it merges like this (ie main was at 0038 and you also had 0038 on your local for a migration you made). I believe you can get past the merging of migration numbers clashing by rolling back the migrations back to the most recent migration in main and then rerunning migrations command again so the changes you made are reflected -- feel free to ping me and I can find the commands and do it with you if it's confusing!

Reply:
A migration merge is one of the relatively easier ways to fix a blown sandbox, but I do get your point. Merges are harder to read. I'll work on it tomorrow.

Reply:
And I'll ping you. Migrations...